api/rbdigital.py

```diff
@@ -1668,6 +1866,10 @@ def fulfill_part(self, part):
         :return: A FulfillmentInfo if the part could be fulfilled;
             a ProblemDetail otherwise.
         """
+        # TODO: This test is needed until we can convert this kwarg into an arg.
+        # if fulfillment_request is None:
+        #     raise ValueError("fulfillment_request parameter must be specified")
+
```

Comment:
This comment seems like a relic of an earlier draft of this branch.

Reply:
Good catch! Thanks.